### ISSUE TITLE
/orange/ versus /blue storage

### ISSUE THREAD

**Issue by bw4sz** [weecology/EvergladesTools#148]
While looking for our latest predictions, I got confused about the location of objects on /orange versus /blue. I _think__  that everything on /orange can be deleted, the full pipeline runs on /blue? @henrykironde?

**Comment by bw4sz** [comment]
Wait for @ethanwhite to return before deleting, but I think we should clean this up.

**Comment by henrykironde** [comment]
The Snake pipeline runs on blue https://github.com/weecology/EvergladesTools/blob/main/Zooniverse/Snakefile

**Comment by henrykironde** [comment]
Check the reference https://wiki.weecology.org/docs/computers-and-programming/hipergator-reference

**Comment by ethanwhite** [comment]
TLDR; We can get rid of `orange` but only after a careful check to make sure anything from earlier work that we need to keep gets moved to `blue`. Most of the volume is in directories that should only need a very quick check: `2021`, `WadingBirds2020`, `utm_projected`, and `mapbox`.

The current workflow is not using anything on `orange`. It would be worth just looking through what is stored there quickly just in case there is something useful there that doesn't have a copy on `blue`. I think that needs to be done by @bw4sz since I've done all of my work following the move to `blue` so I wouldn't recognize anything relevant.

There is still some code in the repo that references `orange`. This is not code that is currently being run by the workflow system, but it's possible we may want some of it in the future. The relevant paths should be updated to `blue`. Most of this is just find/replace (but we do use the word "orange" in non-path contexts, so a little  care is warranted. 

In additional all of the old Zooniverse related stuff, both code and csv files, references `orange`. _This is the place where there is most likely to be meaningful code/data on orange that needs to be moved before deletion.

**Comment by ethanwhite** [comment]
Closing, but feel free to reopen if still needed.
